src/DocTests.jl

```diff
@@ -242,6 +242,8 @@ end
 function eval_repl(block, sandbox, meta::Dict, doc::Documenter.Document, page)
     for (input, output) in repl_splitter(block.code)
         result = Result(block, input, output, meta[:CurrentFile])
+        src_lines = Documenter.find_block_in_file(result.block.code, result.file)
```

Comment:
Ideally, this would be more like
```
src_lines = Documenter.find_block_in_file(input, result.file)
```
to narrow down the exact line, but `find_block_in_file` finds the first match in the file, so lines that appear in more than one doctest are wrong. E.g. initial `using` statements.  As written, there may be slightly confusion in the same line appears twice in one doctest or if one doctest appears as a subset of another.

Reply:
`result.block` should be the same as `block`, so maybe we can move this outside of the `for` loop? In other words, it's looking for the line range of the whole `jldoctest` code block anyway, right?